examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1524,8 +1524,8 @@ class GLTFWriter {
 
 		} else {
 
-			materialDef.pbrMetallicRoughness.metallicFactor = 0.5;
-			materialDef.pbrMetallicRoughness.roughnessFactor = 0.5;
+			materialDef.pbrMetallicRoughness.metallicFactor = 0;
+			materialDef.pbrMetallicRoughness.roughnessFactor = 1;
```

Comment:
Minor, but the [default is `1`](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#_material_pbrmetallicroughness_metallicfactor), so we could omit `roughnessFactor` if we wanted more concise output.

Reply:
I've changed the value of `metallicFactor` to `1` so it matches the glTF spec.

What do you mean with "omit roughnessFactor"?

Reply:
Oops sorry, I don't mean to say that glTF's defaults are better than those you'd chosen. I agree metal=0,rough=1 is likely to be a better representation of phong and lambert materials for example. These glTF defaults were mainly chosen, AFAIK, so that exporters writing metal/rough textures would not be forced to also override the default factors, which isn't relevant to us here.

My only concern was that specifying default values explicitly is redundant, we could just write...

```javascript
materialDef.pbrMetallicRoughness.metallicFactor = 0; // default roughness is 1
```

... and nothing else.

Reply:
Understood!

Reply:
Sorry, [I reverted this](https://github.com/mrdoob/three.js/commit/48e6e2ee4e769013b911a079a433aecc2686d5b2). I feel like the redundant code is more clear.